.github/workflows/ci.yml

```
@@ -12,12 +13,22 @@ concurrency:
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
 permissions:
   contents: read
 
 env:
   TQDM_DISABLE: "1"
   FORCE_COLOR: "1"
+  FORCE_COLOR: "1"
```

actionlint (1.7.12)

[error] key "FORCE_COLOR" is duplicated in "env" section. previously defined at line:30,col:3. note that this key is case insensitive

(syntax-check)